dbt/include/athena/macros/utils/timestamps.sql

```diff
@@ -1,5 +1,15 @@
+{%
+    pyathena converts time zoned timestamps to strings so lets avoid them now()
+%}
```

Comment:
what is the benefit ot using this notation versus in macro comments? `-- pyathena converts time zoned timestamps to strings so lets avoid them`

Reply:
When running in debug mode, the comments made via -- show up in the debug output and I wanted to avoid that. 

Example:

```
, to_hex(md5(to_utf8(coalesce(cast(id as varchar ), '')
         || '|' || coalesce(cast(
    -- pyathena converts time zoned timestamps to strings so lets avoid them
    -- now()
    cast(now() as timestamp)
 as varchar ), '')
        ))) AS dbt_scd_id
```